src/content/docs/errors-inbox/mobile-tab.mdx

```diff
@@ -0,0 +1,260 @@
+* Enable `MobileRequest` events for capturing network request data.
+
+<img
+    title="group crashes event trail"
+    alt="A screenshot showing the event trail for group crashes in the UI"
+    src={errorsinboxEventTrail}
+/>
+
+<figcaption>
+    INSERT_CAPTION_HERE
+</figcaption>
+
+## Mobile stack trace [#stack-trace]
+
+The stack trace for the crashing thread is displayed beneath the crash details. Each frame within the stack indicates the associated library/package name and method. The agent attempts to highlight the specific stack frame containing the programming error responsible for the crash, if detected.
+
+Understanding Stack Traces:
+
+    * **iOS apps**: Source code method names and line numbers are visible if you have uploaded the corresponding dSYM files for each build.
+    * **Android apps using ProGuard**: Package and method names will be de-obfuscated if ProGuard map uploading is enabled.
+    * **Android native crash reports**: These reports will only display source filenames and relative code offsets.
+
+[Email alerts](/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/mobile-monitoring-email-notifications/) promptly inform the team of new crashes. 
+
+## Android-native crash reporting [#android-native-reporting]
+
+Starting with [New Relic Android agent version 6.7.0](/docs/release-notes/mobile-release-notes/android-release-notes/android-670), to help track and diagnose native crashes, reporting, and analysis has been enhanced to include signal violations and other faults that occur at the [native code](https://developer.android.com/ndk/guides) level during runtime.
+
+These enhancements include:
+
+   * Native crash reports - signal violations and other crashes reported by the app during runtime, including
+  * Signal 4 (Illegal instruction)
+  * Signal 6 (Abnormal termination)
+  * Signal 7 (Bus error/bad memory access)
+  * Signal 8 (Floating-point exception)
+  * Signal 11 (Segmentation violation/invalid memory reference)
+
+* Native runtime exceptions
+The native agent will report any unhandled C++ exceptions thrown by the app during runtime, and report them as [handled exceptions](/docs/mobile-monitoring/mobile-monitoring-ui/crashes/introduction-mobile-handled-exceptions/). Unhandled exceptions are usually fatal and will crash the application.
+
+* Application Not Responding (ANR) conditions
+The native agent will detect and report [Application Not Responding](https://developer.android.com/topic/performance/vitals/anr) conditions where an Activity or service thread has been blocked for longer than Android's allowable time, which is 5 seconds for foreground activities, and 5 to 200 seconds for services. ANR conditions will be reported as a [handled exception](/docs/mobile-monitoring/mobile-monitoring-ui/crashes/introduction-mobile-handled-exceptions/) (rather than crashes), since these are not considered fatal conditions.
```

Comment:
Why `Activity` is in uppercase here? This paragraph include passive sentences.

Reply:
Thanks, @nbaenam - this was all copied from another doc that I didn't write. Let me try to rework it. Thanks for calling it out!